src/main/scala/com/mwz/sonar/scala/scapegoat/ScapegoatRulesRepository.scala

```diff
@@ -0,0 +1,50 @@
+package com.mwz.sonar.scala
+package scapegoat
+
+import inspections.{Level, ScapegoatInspection}
+
+import org.sonar.api.server.rule.RulesDefinition
+import org.sonar.api.rule.RuleStatus
+import org.sonar.api.rule.Severity
+import org.sonar.api.rules.RuleType
+
+/** Defines a rules repository for the Scapegoat inspections */
+class ScapegoatRulesRepository extends RulesDefinition {
+
+  /** Defines the rules in the repository */
+  override def define(context: RulesDefinition.Context): Unit = {
+    // crete an empty repository
+    val repository =
+      context
+        .createRepository(ScapegoatRulesRepository.RepositoryKey, Scala.LanguageKey)
+        .setName(ScapegoatRulesRepository.RepositoryName)
+
+    // register each scapegoat inspection as a repository rule
+    for (inspection <- ScapegoatInspection.AllScapegoatInspections) {
+      val rule = repository.createRule(inspection.id)
+      val ruleSeverity = ScapegoatRulesRepository.scapegoatLevelToRuleSeverity(inspection.defaultLevel)
+
+      rule.setInternalKey(inspection.id)
+      rule.setName(s"scala:scalastyle:${inspection.name}")
```

Comment:
This sets the display name, so we probably want it to be just `inspection.name`.

Reply:
Uhm.. to be honest, I really liked this naming convention because it makes easier to search for rules, and it made clear who provided that rule.

Reply:
Just to be clear that this is the display name that we're talking about, so this is how it's going to show in the rules repository e.g. `scala:scoverage:checks for a % 2 == 1 for oddness because this fails on negative numbers`.

There is a nice filtering panel on the left hand side, so if the user want to filter rules by e.g. Scala language or Scapegoat repository they can just select those and get the rules filtered quite easily without seeing any of those prefixes in the names. I haven't seen any other rules having any prefixes as a part of the display name - are you really sure about this?

Reply:
uhm... yes, you're right. It looks awkward.
I'm changing it right now, so I will be pushing the changes in a couple of minutes.